LoopKit/DeviceManager/DeviceManager.swift

```diff
@@ -8,20 +8,25 @@
 import Foundation
 import UserNotifications
 
-public protocol DeviceManagerDelegate {
+public protocol DeviceManagerDelegate: UserAlertHandler {
+    
+    #if !USE_NEW_ALERT_FACILITY
```

Comment:
To make it easier to merge this, I've left the old API, and its usage, in place with this `USE_NEW_ALERT_FACILITY` marker for easy finding

Reply:
Would prefer to keep unused/testing code out of dev, but if it's really too hard to change everything at once, it's ok as long as there is a concrete and short timeline for removal.

Reply:
Yes.  I want to get this into `dev`, please, then post a PR for `DashKit` and Dexcom, then merge those, and then remove this code.  I am hoping by the end of the week...